roles/nvidia_smi/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: check for nvidia_smi
+  stat:
+    path: /usr/bin/nvidia-smi
+  register: nvidia_smi_path
+  ignore_errors: yes
+
+
+- block:
+  - name: capture nvidia_smi info
+    command: nvidia-smi -q -x
```

Comment:
Can you post the typical stdout and stderr of this command on a system with an nvidia GPU please?

Reply:
"stderr": "",

"stdout": "<?xml version=\"1.0\" ?>\n<!DOCTYPE nvidia_smi_log SYSTEM \"nvsmi_device_v10.dtd\">\n<nvidia_smi_log>\n\t<timestamp>Wed Jul  8 10:58:24 2020</timestamp>\n\t<driver_version>440.64.00</driver_version>\n\t<cuda_version>10.2</cuda_version>\n\t<attached_gpus>2</attached_gpus>\n\t<gpu id=\"00000000:02:00.0\">\n\t\t<product_name>Tesla P100-PCIE-16GB</product_name>\n\t\t<product_brand>Tesla</product_brand>\n\t\t<display_mode>Enabled</display_mode>\n\t\t<display_active>Disabled</display_active>\n\t\t<persistence_mode>Disabled</persistence_mode>\n\t\t<accounting_mode>Disabled</accounting_mode>\n\t\t<accounting_mode_buffer_size>4000</accounting_mode_buffer_size>\n\t\t<driver_model>\n\t\t\t<current_dm>N/A</current_dm>\n\t\t\t<pending_dm>N/A</pending_dm>\n\t\t</driver_model>\n\t\t<serial>0324116148444</serial>\n\t\t<uuid>GPU-76b2439c-1cf7-9cea-2424-ccf03501a26b</uuid>\n\t\t<minor_number>0</minor_number>\n\t\t<vbios_version>86.00.26.00.01</vbios_version>\n\t\t<multigpu_board>No</multigpu_board>\n\t\t<board_id>0x200</board_id>\n\t\t<gpu_part_number>900-2H400-0000-000</gpu_part_number>\n\t\t<inforom_version>\n\t\t\t<img_version>H400.0201.00.06</img_version>\n\t\t\t<oem_object>1.1</oem_object>\n\t\t\t<ecc_object>4.1</ecc_object>\n\t\t\t<pwr_object>N/A</pwr_object>\n\t\t</inforom_version>\n\t\t<gpu_operation_mode>\n\t\t\t<current_gom>N/A</current_gom>\n\t\t\t<pending_gom>N/A</pending_gom>\n\t\t</gpu_operation_mode>\n\t\t<gpu_virtualization_mode>\n\t\t\t<virtualization_mode>None</virtualization_mode>\n\t\t\t<host_vgpu_mode>N/A</host_vgpu_mode>\n\t\t</gpu_virtualization_mode>\n\t\t<ibmnpu>\n\t\t\t<relaxed_ordering_mode>N/A</relaxed_ordering_mode>\n\t\t</ibmnpu>\n\t\t<pci>\n\t\t\t<pci_bus>02</pci_bus>\n\t\t\t<pci_device>00</pci_device>\n\t\t\t<pci_domain>0000</pci_domain>\n\t\t\t<pci_device_id>15F810DE</pci_device_id>\n\t\t\t<pci_bus_id>00000000:02:00.0</pci_bus_id>\n\t\t\t<pci_sub_system_id>118F10DE</pci_sub_system_id>\n\t\t\t<pci_gpu_link_info>\n\t\t\t\t<pcie_gen>\n\t\t\t\t\t<max_link_gen>3</max_link_gen>\n\t\t\t\t\t<current_link_gen>3</current_link_gen>\n\t\t\t\t</pcie_gen>\n\t\t\t\t<link_widths>\n\t\t\t\t\t<max_link_width>16x</max_link_width>\n\t\t\t\t\t<current_link_width>16x</current_link_width>\n\t\t\t\t</link_widths>\n\t\t\t</pci_gpu_link_info>\n\t\t\t<pci_bridge_chip>\n\t\t\t\t<bridge_chip_type>N/A</bridge_chip_type>\n\t\t\t\t<bridge_chip_fw>N/A</bridge_chip_fw>\n\t\t\t</pci_bridge_chip>\n\t\t\t<replay_counter>0</replay_counter>\n\t\t\t<replay_rollover_counter>0</replay_rollover_counter>\n\t\t\t<tx_util>0 KB/s</tx_util>\n\t\t\t<rx_util>0 KB/s</rx_util>\n\t\t</pci>\n\t\t<fan_speed>N/A</fan_speed>\n\t\t<performance_state>P0</performance_state>\n\t\t<clocks_throttle_reasons>\n\t\t\t<clocks_throttle_reason_gpu_idle>Not Active</clocks_throttle_reason_gpu_idle>\n\t\t\t<clocks_throttle_reason_applications_clocks_setting>Not Active</clocks_throttle_reason_applications_clocks_setting>\n\t\t\t<clocks_throttle_reason_sw_power_cap>Not Active</clocks_throttle_reason_sw_power_cap>\n\t\t\t<clocks_throttle_reason_hw_slowdown>Not Active</clocks_throttle_reason_hw_slowdown>\n\t\t\t<clocks_throttle_reason_hw_thermal_slowdown>Not Active</clocks_throttle_reason_hw_thermal_slowdown>\n\t\t\t<clocks_throttle_reason_hw_power_brake_slowdown>Not Active</clocks_throttle_reason_hw_power_brake_slowdown>\n\t\t\t<clocks_throttle_reason_sync_boost>Not Active</clocks_throttle_reason_sync_boost>\n\t\t\t<clocks_throttle_reason_sw_thermal_slowdown>Not Active</clocks_throttle_reason_sw_thermal_slowdown>\n\t\t\t<clocks_throttle_reason_display_clocks_setting>Not Active</clocks_throttle_reason_display_clocks_setting>\n\t\t</clocks_throttle_reasons>\n\t\t<fb_memory_usage>\n\t\t\t<total>16280 MiB</total>\n\t\t\t<used>0 MiB</used>\n\t\t\t<free>16280 MiB</free>\n\t\t</fb_memory_usage>\n\t\t<bar1_memory_usage>\n\t\t\t<total>16384 MiB</total>\n\t\t\t<used>2 MiB</used>\n\t\t\t<free>16382 MiB</free>\n\t\t</bar1_memory_usage>\n\t\t<compute_mode>Default</compute_mode>\n\t\t<utilization>\n\t\t\t<gpu_util>0 %</gpu_util>\n\t\t\t<memory_util>0 %</memory_util>\n\t\t\t<encoder_util>0 %</encoder_util>\n\t\t\t<decoder_util>0 %</decoder_util>\n\t\t</utilization>\n\t\t<encoder_stats>\n\t\t\t<session_count>0</session_count>\n\t\t\t<average_fps>0</average_fps>\n\t\t\t<average_latency>0</average_latency>\n\t\t</encoder_stats>\n\t\t<fbc_stats>\n\t\t\t<session_count>0</session_count>\n\t\t\t<average_fps>0</average_fps>\n\t\t\t<average_latency>0</average_latency>\n\t\t</fbc_stats>\n\t\t<ecc_mode>\n\t\t\t<current_ecc>Enabled</current_ecc>\n\t\t\t<pending_ecc>Enabled</pending_ecc>\n\t\t</ecc_mode>\n\t\t<ecc_errors>\n\t\t\t<volatile>\n\t\t\t\t<single_bit>\n\t\t\t\t\t<device_memory>0</device_memory>\n\t\t\t\t\t<register_file>0</register_file>\n\t\t\t\t\t<l1_cache>N/A</l1_cache>\n\t\t\t\t\t<l2_cache>0</l2_cache>\n\t\t\t\t\t<texture_memory>0</texture_memory>\n\t\t\t\t\t<texture_shm>0</texture_shm>\n\t\t\t\t\t<cbu>N/A</cbu>\n\t\t\t\t\t<total>0</total>\n\t\t\t\t</single_bit>\n\t\t\t\t<double_bit>\n\t\t\t\t\t<device_memory>0</device_memory>\n\t\t\t\t\t<register_file>0</register_file>\n\t\t\t\t\t<l1_cache>N/A</l1_cache>\n\t\t\t\t\t<l2_cache>0</l2_cache>\n\t\t\t\t\t<texture_memory>0</texture_memory>\n\t\t\t\t\t<texture_shm>0</texture_shm>\n\t\t\t\t\t<cbu>N/A</cbu>\n\t\t\t\t\t<total>0</total>\n\t\t\t\t</double_bit>\n\t\t\t</volatile>\n\t\t\t<aggregate>\n\t\t\t\t<single_bit>\n\t\t\t\t\t<device_memory>0</device_memory>\n\t\t\t\t\t<register_file>0</register_file>\n\t\t\t\t\t<l1_cache>N/A</l1_cache>\n\t\t\t\t\t<l2_cache>0</l2_cache>\n\t\t\t\t\t<texture_memory>0</texture_memory>\n\t\t\t\t\t<texture_shm>0</texture_shm>\n\t\t\t\t\t<cbu>N/A</cbu>\n\t\t\t\t\t<total>0</total>\n\t\t\t\t</single_bit>\n\t\t\t\t<double_bit>\n\t\t\t\t\t<device_memory>0</device_memory>\n\t\t\t\t\t<register_file>0</register_file>\n\t\t\t\t\t<l1_cache>N/A</l1_cache>\n\t\t\t\t\t<l2_cache>0</l2_cache>\n\t\t\t\t\t<texture_memory>0</texture_memory>\n\t\t\t\t\t<texture_shm>0</texture_shm>\n\t\t\t\t\t<cbu>N/A</cbu>\n\t\t\t\t\t<total>0</total>\n\t\t\t\t</double_bit>\n\t\t\t</aggregate>\n\t\t</ecc_errors>\n\t\t<retired_pages>\n\t\t\t<multiple_single_bit_retirement>\n\t\t\t\t<retired_count>0</retired_count>\n\t\t\t\t<retired_pagelist>\n\t\t\t\t</retired_pagelist>\n\t\t\t</multiple_single_bit_retirement>\n\t\t\t<double_bit_retirement>\n\t\t\t\t<retired_count>0</retired_count>\n\t\t\t\t<retired_pagelist>\n\t\t\t\t</retired_pagelist>\n\t\t\t</double_bit_retirement>\n\t\t\t<pending_blacklist>No</pending_blacklist>\n\t\t\t<pending_retirement>No</pending_retirement>\n\t\t</retired_pages>\n\t\t<temperature>\n\t\t\t<gpu_temp>38 C</gpu_temp>\n\t\t\t<gpu_temp_max_threshold>85 C</gpu_temp_max_threshold>\n\t\t\t<gpu_temp_slow_threshold>82 C</gpu_temp_slow_threshold>\n\t\t\t<gpu_temp_max_gpu_threshold>N/A</gpu_temp_max_gpu_threshold>\n\t\t\t<memory_temp>N/A</memory_temp>\n\t\t\t<gpu_temp_max_mem_threshold>N/A</gpu_temp_max_mem_threshold>\n\t\t</temperature>\n\t\t<power_readings>\n\t\t\t<power_state>P0</power_state>\n\t\t\t<power_management>Supported</power_management>\n\t\t\t<power_draw>35.85 W</power_draw>\n\t\t\t<power_limit>250.00 W</power_limit>\n\t\t\t<default_power_limit>250.00 W</default_power_limit>\n\t\t\t<enforced_power_limit>250.00 W</enforced_power_limit>\n\t\t\t<min_power_limit>125.00 W</min_power_limit>\n\t\t\t<max_power_limit>250.00 W</max_power_limit>\n\t\t</power_readings>\n\t\t<clocks>\n\t\t\t<graphics_clock>1189 MHz</graphics_clock>\n\t\t\t<sm_clock>1189 MHz</sm_clock>\n\t\t\t<mem_clock>715 MHz</mem_clock>\n\t\t\t<video_clock>1063 MHz</video_clock>\n\t\t</clocks>\n\t\t<applications_clocks>\n\t\t\t<graphics_clock>1189 MHz</graphics_clock>\n\t\t\t<mem_clock>715 MHz</mem_clock>\n\t\t</applications_clocks>\n\t\t<default_applications_clocks>\n\t\t\t<graphics_clock>1189 MHz</graphics_clock>\n\t\t\t<mem_clock>715 MHz</mem_clock>\n\t\t</default_applications_clocks>\n\t\t<max_clocks>\n\t\t\t<graphics_clock>1328 MHz</graphics_clock>\n\t\t\t<sm_clock>1328 MHz</sm_clock>\n\t\t\t<mem_clock>715 MHz</mem_clock>\n\t\t\t<video_clock>1328 MHz</video_clock>\n\t\t</max_clocks>\n\t\t<max_customer_boost_clocks>\n\t\t\t<graphics_clock>N/A</graphics_clock>\n\t\t</max_customer_boost_clocks>\n\t\t<clock_policy>\n\t\t\t<auto_boost>N/A</auto_boost>\n\t\t\t<auto_boost_default>N/A</auto_boost_default>\n\t\t</clock_policy>\n\t\t<supported_clocks>\n\t\t\t<supported_mem_clock>\n\t\t\t\t<value>715 MHz</value>\n\t\t\t\t<supported_graphics_clock>1328 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1316 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1303 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1290 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1278 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1265 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1252 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1240 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1227 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1215 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1202 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1189 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1177 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1164 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1151 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1139 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1126 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1113 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1101 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1088 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1075 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1063 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1050 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1037 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1025 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1012 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>999 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>987 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>974 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>961 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>949 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>936 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>923 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>911 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>898 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>885 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>873 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>860 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>847 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>835 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>822 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>810 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>797 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>784 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>772 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>759 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>746 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>734 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>721 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>708 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>696 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>683 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>670 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>658 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>645 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>632 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>620 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>607 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>594 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>582 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>569 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>556 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>544 MHz</supported_graphics_clock>\n\t\t\t</supported_mem_clock>\n\t\t</supported_clocks>\n\t\t<processes>\n\t\t</processes>\n\t\t<accounted_processes>\n\t\t</accounted_processes>\n\t</gpu>\n\n\t<gpu id=\"00000000:81:00.0\">\n\t\t<product_name>Tesla P100-PCIE-16GB</product_name>\n\t\t<product_brand>Tesla</product_brand>\n\t\t<display_mode>Enabled</display_mode>\n\t\t<display_active>Disabled</display_active>\n\t\t<persistence_mode>Disabled</persistence_mode>\n\t\t<accounting_mode>Disabled</accounting_mode>\n\t\t<accounting_mode_buffer_size>4000</accounting_mode_buffer_size>\n\t\t<driver_model>\n\t\t\t<current_dm>N/A</current_dm>\n\t\t\t<pending_dm>N/A</pending_dm>\n\t\t</driver_model>\n\t\t<serial>0324116148443</serial>\n\t\t<uuid>GPU-7e43d4c3-7386-79cb-9540-213005972ef4</uuid>\n\t\t<minor_number>1</minor_number>\n\t\t<vbios_version>86.00.26.00.01</vbios_version>\n\t\t<multigpu_board>No</multigpu_board>\n\t\t<board_id>0x8100</board_id>\n\t\t<gpu_part_number>900-2H400-0000-000</gpu_part_number>\n\t\t<inforom_version>\n\t\t\t<img_version>H400.0201.00.06</img_version>\n\t\t\t<oem_object>1.1</oem_object>\n\t\t\t<ecc_object>4.1</ecc_object>\n\t\t\t<pwr_object>N/A</pwr_object>\n\t\t</inforom_version>\n\t\t<gpu_operation_mode>\n\t\t\t<current_gom>N/A</current_gom>\n\t\t\t<pending_gom>N/A</pending_gom>\n\t\t</gpu_operation_mode>\n\t\t<gpu_virtualization_mode>\n\t\t\t<virtualization_mode>None</virtualization_mode>\n\t\t\t<host_vgpu_mode>N/A</host_vgpu_mode>\n\t\t</gpu_virtualization_mode>\n\t\t<ibmnpu>\n\t\t\t<relaxed_ordering_mode>N/A</relaxed_ordering_mode>\n\t\t</ibmnpu>\n\t\t<pci>\n\t\t\t<pci_bus>81</pci_bus>\n\t\t\t<pci_device>00</pci_device>\n\t\t\t<pci_domain>0000</pci_domain>\n\t\t\t<pci_device_id>15F810DE</pci_device_id>\n\t\t\t<pci_bus_id>00000000:81:00.0</pci_bus_id>\n\t\t\t<pci_sub_system_id>118F10DE</pci_sub_system_id>\n\t\t\t<pci_gpu_link_info>\n\t\t\t\t<pcie_gen>\n\t\t\t\t\t<max_link_gen>3</max_link_gen>\n\t\t\t\t\t<current_link_gen>3</current_link_gen>\n\t\t\t\t</pcie_gen>\n\t\t\t\t<link_widths>\n\t\t\t\t\t<max_link_width>16x</max_link_width>\n\t\t\t\t\t<current_link_width>16x</current_link_width>\n\t\t\t\t</link_widths>\n\t\t\t</pci_gpu_link_info>\n\t\t\t<pci_bridge_chip>\n\t\t\t\t<bridge_chip_type>N/A</bridge_chip_type>\n\t\t\t\t<bridge_chip_fw>N/A</bridge_chip_fw>\n\t\t\t</pci_bridge_chip>\n\t\t\t<replay_counter>0</replay_counter>\n\t\t\t<replay_rollover_counter>0</replay_rollover_counter>\n\t\t\t<tx_util>1000 KB/s</tx_util>\n\t\t\t<rx_util>3000 KB/s</rx_util>\n\t\t</pci>\n\t\t<fan_speed>N/A</fan_speed>\n\t\t<performance_state>P0</performance_state>\n\t\t<clocks_throttle_reasons>\n\t\t\t<clocks_throttle_reason_gpu_idle>Not Active</clocks_throttle_reason_gpu_idle>\n\t\t\t<clocks_throttle_reason_applications_clocks_setting>Not Active</clocks_throttle_reason_applications_clocks_setting>\n\t\t\t<clocks_throttle_reason_sw_power_cap>Not Active</clocks_throttle_reason_sw_power_cap>\n\t\t\t<clocks_throttle_reason_hw_slowdown>Not Active</clocks_throttle_reason_hw_slowdown>\n\t\t\t<clocks_throttle_reason_hw_thermal_slowdown>Not Active</clocks_throttle_reason_hw_thermal_slowdown>\n\t\t\t<clocks_throttle_reason_hw_power_brake_slowdown>Not Active</clocks_throttle_reason_hw_power_brake_slowdown>\n\t\t\t<clocks_throttle_reason_sync_boost>Not Active</clocks_throttle_reason_sync_boost>\n\t\t\t<clocks_throttle_reason_sw_thermal_slowdown>Not Active</clocks_throttle_reason_sw_thermal_slowdown>\n\t\t\t<clocks_throttle_reason_display_clocks_setting>Not Active</clocks_throttle_reason_display_clocks_setting>\n\t\t</clocks_throttle_reasons>\n\t\t<fb_memory_usage>\n\t\t\t<total>16280 MiB</total>\n\t\t\t<used>0 MiB</used>\n\t\t\t<free>16280 MiB</free>\n\t\t</fb_memory_usage>\n\t\t<bar1_memory_usage>\n\t\t\t<total>16384 MiB</total>\n\t\t\t<used>2 MiB</used>\n\t\t\t<free>16382 MiB</free>\n\t\t</bar1_memory_usage>\n\t\t<compute_mode>Default</compute_mode>\n\t\t<utilization>\n\t\t\t<gpu_util>4 %</gpu_util>\n\t\t\t<memory_util>0 %</memory_util>\n\t\t\t<encoder_util>0 %</encoder_util>\n\t\t\t<decoder_util>0 %</decoder_util>\n\t\t</utilization>\n\t\t<encoder_stats>\n\t\t\t<session_count>0</session_count>\n\t\t\t<average_fps>0</average_fps>\n\t\t\t<average_latency>0</average_latency>\n\t\t</encoder_stats>\n\t\t<fbc_stats>\n\t\t\t<session_count>0</session_count>\n\t\t\t<average_fps>0</average_fps>\n\t\t\t<average_latency>0</average_latency>\n\t\t</fbc_stats>\n\t\t<ecc_mode>\n\t\t\t<current_ecc>Enabled</current_ecc>\n\t\t\t<pending_ecc>Enabled</pending_ecc>\n\t\t</ecc_mode>\n\t\t<ecc_errors>\n\t\t\t<volatile>\n\t\t\t\t<single_bit>\n\t\t\t\t\t<device_memory>0</device_memory>\n\t\t\t\t\t<register_file>0</register_file>\n\t\t\t\t\t<l1_cache>N/A</l1_cache>\n\t\t\t\t\t<l2_cache>0</l2_cache>\n\t\t\t\t\t<texture_memory>0</texture_memory>\n\t\t\t\t\t<texture_shm>0</texture_shm>\n\t\t\t\t\t<cbu>N/A</cbu>\n\t\t\t\t\t<total>0</total>\n\t\t\t\t</single_bit>\n\t\t\t\t<double_bit>\n\t\t\t\t\t<device_memory>0</device_memory>\n\t\t\t\t\t<register_file>0</register_file>\n\t\t\t\t\t<l1_cache>N/A</l1_cache>\n\t\t\t\t\t<l2_cache>0</l2_cache>\n\t\t\t\t\t<texture_memory>0</texture_memory>\n\t\t\t\t\t<texture_shm>0</texture_shm>\n\t\t\t\t\t<cbu>N/A</cbu>\n\t\t\t\t\t<total>0</total>\n\t\t\t\t</double_bit>\n\t\t\t</volatile>\n\t\t\t<aggregate>\n\t\t\t\t<single_bit>\n\t\t\t\t\t<device_memory>0</device_memory>\n\t\t\t\t\t<register_file>0</register_file>\n\t\t\t\t\t<l1_cache>N/A</l1_cache>\n\t\t\t\t\t<l2_cache>0</l2_cache>\n\t\t\t\t\t<texture_memory>0</texture_memory>\n\t\t\t\t\t<texture_shm>0</texture_shm>\n\t\t\t\t\t<cbu>N/A</cbu>\n\t\t\t\t\t<total>0</total>\n\t\t\t\t</single_bit>\n\t\t\t\t<double_bit>\n\t\t\t\t\t<device_memory>0</device_memory>\n\t\t\t\t\t<register_file>0</register_file>\n\t\t\t\t\t<l1_cache>N/A</l1_cache>\n\t\t\t\t\t<l2_cache>0</l2_cache>\n\t\t\t\t\t<texture_memory>0</texture_memory>\n\t\t\t\t\t<texture_shm>0</texture_shm>\n\t\t\t\t\t<cbu>N/A</cbu>\n\t\t\t\t\t<total>0</total>\n\t\t\t\t</double_bit>\n\t\t\t</aggregate>\n\t\t</ecc_errors>\n\t\t<retired_pages>\n\t\t\t<multiple_single_bit_retirement>\n\t\t\t\t<retired_count>0</retired_count>\n\t\t\t\t<retired_pagelist>\n\t\t\t\t</retired_pagelist>\n\t\t\t</multiple_single_bit_retirement>\n\t\t\t<double_bit_retirement>\n\t\t\t\t<retired_count>0</retired_count>\n\t\t\t\t<retired_pagelist>\n\t\t\t\t</retired_pagelist>\n\t\t\t</double_bit_retirement>\n\t\t\t<pending_blacklist>No</pending_blacklist>\n\t\t\t<pending_retirement>No</pending_retirement>\n\t\t</retired_pages>\n\t\t<temperature>\n\t\t\t<gpu_temp>44 C</gpu_temp>\n\t\t\t<gpu_temp_max_threshold>85 C</gpu_temp_max_threshold>\n\t\t\t<gpu_temp_slow_threshold>82 C</gpu_temp_slow_threshold>\n\t\t\t<gpu_temp_max_gpu_threshold>N/A</gpu_temp_max_gpu_threshold>\n\t\t\t<memory_temp>N/A</memory_temp>\n\t\t\t<gpu_temp_max_mem_threshold>N/A</gpu_temp_max_mem_threshold>\n\t\t</temperature>\n\t\t<power_readings>\n\t\t\t<power_state>P0</power_state>\n\t\t\t<power_management>Supported</power_management>\n\t\t\t<power_draw>30.77 W</power_draw>\n\t\t\t<power_limit>250.00 W</power_limit>\n\t\t\t<default_power_limit>250.00 W</default_power_limit>\n\t\t\t<enforced_power_limit>250.00 W</enforced_power_limit>\n\t\t\t<min_power_limit>125.00 W</min_power_limit>\n\t\t\t<max_power_limit>250.00 W</max_power_limit>\n\t\t</power_readings>\n\t\t<clocks>\n\t\t\t<graphics_clock>1050 MHz</graphics_clock>\n\t\t\t<sm_clock>1050 MHz</sm_clock>\n\t\t\t<mem_clock>715 MHz</mem_clock>\n\t\t\t<video_clock>949 MHz</video_clock>\n\t\t</clocks>\n\t\t<applications_clocks>\n\t\t\t<graphics_clock>1189 MHz</graphics_clock>\n\t\t\t<mem_clock>715 MHz</mem_clock>\n\t\t</applications_clocks>\n\t\t<default_applications_clocks>\n\t\t\t<graphics_clock>1189 MHz</graphics_clock>\n\t\t\t<mem_clock>715 MHz</mem_clock>\n\t\t</default_applications_clocks>\n\t\t<max_clocks>\n\t\t\t<graphics_clock>1328 MHz</graphics_clock>\n\t\t\t<sm_clock>1328 MHz</sm_clock>\n\t\t\t<mem_clock>715 MHz</mem_clock>\n\t\t\t<video_clock>1328 MHz</video_clock>\n\t\t</max_clocks>\n\t\t<max_customer_boost_clocks>\n\t\t\t<graphics_clock>N/A</graphics_clock>\n\t\t</max_customer_boost_clocks>\n\t\t<clock_policy>\n\t\t\t<auto_boost>N/A</auto_boost>\n\t\t\t<auto_boost_default>N/A</auto_boost_default>\n\t\t</clock_policy>\n\t\t<supported_clocks>\n\t\t\t<supported_mem_clock>\n\t\t\t\t<value>715 MHz</value>\n\t\t\t\t<supported_graphics_clock>1328 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1316 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1303 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1290 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1278 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1265 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1252 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1240 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1227 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1215 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1202 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1189 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1177 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1164 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1151 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1139 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1126 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1113 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1101 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1088 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1075 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1063 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1050 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1037 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1025 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>1012 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>999 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>987 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>974 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>961 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>949 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>936 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>923 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>911 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>898 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>885 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>873 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>860 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>847 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>835 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>822 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>810 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>797 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>784 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>772 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>759 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>746 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>734 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>721 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>708 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>696 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>683 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>670 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>658 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>645 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>632 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>620 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>607 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>594 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>582 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>569 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>556 MHz</supported_graphics_clock>\n\t\t\t\t<supported_graphics_clock>544 MHz</supported_graphics_clock>\n\t\t\t</supported_mem_clock>\n\t\t</supported_clocks>\n\t\t<processes>\n\t\t</processes>\n\t\t<accounted_processes>\n\t\t</accounted_processes>\n\t</gpu>\n\n</nvidia_smi_log>"